packages/studio-plugin/src/orchestrators/ParsingOrchestrator.ts

```diff
@@ -216,12 +227,13 @@ export default class ParsingOrchestrator {
     );
   };
 
-  private updateFilepathToDependencyTree(absPath: string) {
-    this.filepathToDependencyTree[absPath] = dependencyTree({
+  private updateDependencyTrees(absPath: string) {
+    const newDepTree = dependencyTree({
       filename: absPath,
       directory: upath.dirname(absPath),
-      visited: this.filepathToDependencyTree,
+      visited: this.dependencyTrees,
     });
+    this.dependencyTrees = Object.assign(this.dependencyTrees, newDepTree);
```

Comment:
could we do a spread here instead? I personally find that a little more readable

Reply:
done!